### PR TITLE
Extract edit

### PR DESCRIPTION
**Pull request by DKWeeto**
Some extract edits, also fix on lats and longs in metadata

### CODE REVIEW

File: ETL_pipeline/extract.py [hunk 1]
```diff
@@ -6,27 +6,30 @@
 import requests
 
 
+def time_rounder(t):
```
Comment (arj0713) [inline ETL_pipeline/extract.py]: Doc string and type hints

Comment (DKWeeto) [reply ETL_pipeline/extract.py]: 👍

File: ETL_pipeline/extract.py [hunk 2]
```diff
@@ -6,27 +6,42 @@
...
+        today = time_rounder(datetime.now())
+        time_str = today.isoformat(timespec='minutes')
+        tomorrow = time_rounder(today, get_fifteen=False) + timedelta(days=1)
+    else:
+        today = time_rounder(datetime.now(), get_fifteen=False)
+        next_week = today + timedelta(weeks=1)
+        time_str = next_week.isoformat(timespec='minutes')
+        tomorrow = today + timedelta(hours=25)
+    tomorrow_str = tomorrow.isoformat(timespec='minutes')
+    return time_str, tomorrow_str
```
Comment (NMcKittie) [inline ETL_pipeline/extract.py:28]: is there a way to do this without returning 2 variables?

Comment (NMcKittie) [reply ETL_pipeline/extract.py:28]: Its just not good practise to return 2

**Review by NMcKittie — APPROVED** [review]
Looks good

**Pull request merged**